### DISCUSSION TITLE
access to red box in Pick point

### DISCUSSION THREAD

**Discussion by ghazalbangash** [pyvista/pyvista#1436] [Q&A]
Hi
Is there any way for me to have access to the red box around the meshes while Point Picking? I want to remove it but I can't seem to have any access to it (using actor)
![redbox](https://user-images.githubusercontent.com/47847589/115984389-2cdc4580-a5bc-11eb-8568-923fcc79974b.png)

**Comment by banesullivan** [comment]
This is the same as https://github.com/pyvista/pyvista/discussions/1672

This is a known issue. We do not currently have a solution
